packages/suite-build/utils/env.ts

```diff
@@ -11,16 +11,16 @@ const {
 } = process.env;
 
 const project = PROJECT as Project;
-const environment = NODE_ENV === 'production' ? 'production' : 'development';
-const isDev = environment === 'development';
+const mode = NODE_ENV === 'production' ? 'production' : 'development';
+const isDev = mode === 'development';
```

Comment:
let's delete unused `mode`
```suggestion
const isDev = NODE_ENV !== 'production';
```